### PR TITLE
Added description about network discovery

### PR DESCRIPTION
**Pull request by d7oc**
This change is required as the connection doesn't work with Windows 10 out of the box. The changes described here need to be done so the connection can be established.

### CODE REVIEW

**Comment by mmattel** [comment]
Making this a draft for more checking if all stuff is in.

**Comment by d7oc** [comment]
Move windows hints to the troubleshooting section and place xref pointer there.

**Comment by d7oc** [comment]
Add comment that registry changes are only required if the server is connected without SSL

File: modules/classic_ui/pages/files/access_webdav.adoc [hunk 1]
```diff
@@ -451,6 +459,31 @@ sc start "WebClient"
 
 More details can be found https://github.com/owncloud/documentation/pull/2668[here].
 
+=== Problem: Network Discovery
+
+On Windows 10, you need to turn on the _Network Discovery_ to make webdav access working. In newer versions of Windows 10 is enabled but default.
```
Comment (phil-davis) [inline modules/classic_ui/pages/files/access_webdav.adoc:464]:
```suggestion
On Windows 10, you need to turn on the _Network Discovery_ to make webdav access work. In newer versions of Windows 10 it is enabled by default.
```

**Comment by d7oc** [comment]
@mmattel Please check again

**Review by mmattel — APPROVED** [review]

**Comment by mmattel** [comment]
I did some updates/improvements and text fixes, putting to ready for review

**Pull request merged**